src/cli/utils/fixtures/asyncFunctionWithAsyncData.js

```diff
@@ -0,0 +1,22 @@
+async function asyncConfig(){
```

Comment:
This is not necessary, you can just return the config in the export

Reply:
@SGFGOV let me know when you have the time to finalize :) can't wait to merge that

Reply:
sorry,  will try to complete it tomorrow.